cmd/serve/option.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serve
```

Comment:
I wonder, if we are anyways changing the public signature, should we move this into its own package inside `pkg/`? `hubble serve` as a `cmd` will go away once Cilium 1.7 is EOL, so now might be a good time to move this logic.

Reply:
Indeed but as we're about to move all server code into Cilium's git repository anyway, I thought it could be done then. Also, I'd rename `pkg/server` to `pkg/service/observer` (because there will also be `pkg/service/peer`) and would move the code in this PR (minus cobra related things) to `pkg/server` but this should probably be done during code import into Cilium.

Reply:
Sounds good to me!